homework/test/test-list.js

```diff
@@ -1,8 +1,17 @@
 describe('Test > ', function() {
+  var subject;
   beforeEach(function() {
+    subject = new ToDoListManager();
   });
 
   it('will test some pure functions', function() {
     // Write any pure function assertion here.
+    var test = subject;
+    var sum = test.puerFunction(1,4);
+    assert.equal(sum,5,'It is equal to 5!');
   });
+  
+  Manager.prototype.puerFunction(a,b){
```

Comment:
1. There is no `Manager` constructor in this file and the included files
2. Put it in any manager you have defined
3. Are you sure this works well?